frontend/src/components/allocation/list.js

```diff
@@ -1,53 +1,76 @@
 import React, { PropTypes } from 'react';
+import { DropdownButton } from 'react-bootstrap';
 import NomadLink from '../link';
 import FormatTime from '../format/time';
+import shortUUID from '../../helpers/uuid';
 import { renderDesiredStatus, renderClientStatus } from '../../helpers/render/allocation';
 
 const allocationStatusColors = {
-    complete: 'success',
-    running: 'info',
+    complete: '',
+    running: '',
     lost: 'warning',
     failed: 'danger',
 };
 
+const getAllocationNumberFromName = (allocationName) => {
+    const match = /[\d+]/.exec(allocationName);
+    return match[0];
+};
+
 const AllocationList = ({ allocations, nodes }) =>
   <table className="table table-hover table-striped">
     <thead>
       <tr>
+        <th></th>
         <th>ID</th>
         <th>Job</th>
         <th>Task Group</th>
-        <th>Task</th>
+        <th>Client</th>
         <th>Client Status</th>
-        <th>Desired Status</th>
-        <th>Node</th>
-        <th>Evaluation</th>
-        <th>Time</th>
+        <th>When</th>
+        <th></th>
       </tr>
     </thead>
     <tbody>
       {allocations.map((allocation) => {
           const color = allocationStatusColors[allocation.ClientStatus];
           return (
             <tr className={ color } key={ allocation.ID }>
+              <td>{ renderClientStatus(allocation) }</td>
```

Comment:
need to find a better icon then, for me the cog symbolizes something that is running (cogs run around in machines :)) 

its redundant in most cases, but the client status and desired client status can be vastly different when things go wrong

the icon is easy to scan, and in a long list easily allow you to see if something is normal  or not :)